package.json

```diff
@@ -1,10 +1,11 @@
 {
-    "name": "wallet-address-validator",
+    "name": "road2crypto-wallet-address-validator",
     "version": "1.0.0",
     "description": "A simple utility to validate cryptocurrency wallet addresses for EVM, Solana, and Bitcoin chains.",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "scripts": {
```

Comment:
can you also add here the command to publish it, we should be able to do: `npm run publish`

Reply:
done